addon/components/challenges/add.js

```diff
@@ -0,0 +1,23 @@
+import layout from 'bepstore-goals/templates/components/challenges/add';
+import Ember from 'ember';
+import ClickOutside from '../../mixins/click-outside';
+
+const { computed: { equal } } = Ember;
+
+export default Ember.Component.extend(ClickOutside, {
+  layout,
+
+  isRepo: equal('type', 'Repository'),
```

Comment:
The repository model is called `Repo`. I can change it to `Repository`, but `Repo` seems appropriate as well, since the github endpoint is `/repos/:owner/:name`